src/iv/imageviewer.cpp

```diff
@@ -1256,13 +1252,14 @@ ImageViewer::displayCurrentImage(bool update)
 
     if (update) {
         glwin->update();
+        glwin->clamp_view_to_window();
```

Comment:
that what make sure loaded image (when switching between images for example) is in the view. if images is smaller then viewport this will center it as well

Reply:
reverted